qiita_db/test/test_data.py

```diff
@@ -261,6 +261,28 @@ def test_remove_filepath_errors(self):
         with self.assertRaises(QiitaDBError):
             RawData(2).remove_filepath(fp)
 
+    def test_exists(self):
+        self.assertTrue(RawData.exists(1))
+        self.assertFalse(RawData.exists(1000))
+
+    def test_delete(self):
+        # the raw data doesn't exist
+        with self.assertRaises(ValueError):
+            RawData.delete(1000, 1)
+
+        # the raw data and the study id are not linked
+        with self.assertRaises(ValueError):
```

Comment:
In this test is not only that the raw data and the study are not linked, is that the study does not exist. Can we check it with a study that actually exists?